department-of-veterans-affairs/vets-website
# Update list loop aria labels

## Summary
Update list loop aria labels on the Pension Benefits form (21P-527EZ) to improve context for screen reader support.

## Description
The Pension Benefits form list loop pattern button screen reader language is more descriptive by including unique text in the aria labels for screen readers to announce. For example, instead of announcing an 'Edit' button as 'Edit name', it will now include the full name in the announcement, 'Edit John Doe', etc. 

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Turn on a browser screen reader (Voiceover for MacOS)
3. Go to a list loop pattern page and add an item. For example, the 'Add other service names' page of the 'Military information' chapter
4. Verify that the screen reader announces additional context when reading out an item's 'Edit', 'Update' and 'Remove' item buttons

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73969)

## Related issue(s)
n/a

## Testing done

## Screenshots
n/a

## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
